astropy/coordinates/sites.py

```diff
@@ -102,12 +102,15 @@ def from_json(cls, jsondb):
         reg = cls()
         for site in jsondb:
             site_info = jsondb[site]
```

Comment:
This implicitly does a copy? Otherwise the information would be popped away now...

Reply:
Hmm, good point, I guess it doesn't!  In principal it doesn't matter since in all built-in cases the db is only used once I think.  But better safe than sorry!